scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/SortedBucketSource.java

```diff
@@ -434,11 +434,36 @@ public BucketedInput(
         String filenameSuffix,
```

Comment:
Hmm, writing a test for this now, we'll need a way to couple `filenameSuffix` to the specific directory as well